src/frontends/pytorch/src/op/lerp.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/subtract.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_lerp(const NodeContext& context) {
+    num_inputs_check(context, 3, 3);
```

Comment:
could you please stay comment about expected input signature, it will help to track if something will change in future pytorch versions

Reply:
done